zsh-aws-vault.plugin.zsh

```diff
@@ -83,6 +92,9 @@ function _using_osx() {
 }
 
 function _find_browser() {
+  if [ -n "$AWS_VAULT_PL_BROWSER" ]; then
+    echo "$AWS_VAULT_PL_BROWSER"
```

Comment:
Should this be part of the `if` statement below to return immediately if this variable is set?

Reply:
👌